pluribus/games/short_deck/state.py

```diff
@@ -118,7 +118,7 @@ def apply_action(self, action_str: Optional[str]) -> ShortDeckPokerState:
                 f"type {type(action)}."
             )
         # Update the new state.
-        new_state._history.append(action)
+        new_state._history.append((self.player_i, self._betting_stage, action))
```

Comment:
Are we diverging from the behaviour of the original algorithm by appending anything else here, or is it effectively redundant information just to help the human reader debug stuff?

If the former, I'd suggest to revert to just `action`, otherwise if it's the latter, then could I suggest we change this to a format string? Our history would then be a list of strings rather than tuples, and we could make it a little easier to read, i.e
```python
new_state._history.append(f"\nplayer={self.player_i}, betting_stage={self._betting_stage}, action={action}")
```

This will all be amalgamated into the infoset string, which will be massive, but easier to read the history.

Reply:
Excuse my ad hoc debugging code. Just making my changes public. But no, this was just for my debugging purposes. I'm leaning toward this is redundant information, but I am unsure at this point

Reply:
Yea true I have been a bit over-zealous with this PR review, feel free to ignore

Reply:
Haha, no I appreciate it always